printable-recipe-generator/src/tests/main.test.ts

```diff
@@ -3,11 +3,69 @@ import path from 'path';
 import * as ejs from 'ejs';
 import recipe from '../data/sampleRecipe.json';
 
+//load Contributors
+async function getChefs(): Promise<unknown> {
+	const resp = await fetch(
+		'https://recipes.guardianapis.com/v2/contributors.json',
```

Comment:
I would not load this in from server in a test - you only want the test to fail if you have a code error, not a failure on the CDN (say).  Instead, I would just create a simple fixture for your chefs object with only the data you need:

```typescript
const chefs:Record<string,ChefData> = {
  'profile/name': {...manually input data...}
}
```

I would also include test cases for three permutations - one where the chef exists in the list, one where the chef does not exist in the list, and one where the list is `null`.  That way we can protect ourselves against unexpected crashes.

Reply:
O yes, I did this in first place to save writing test data and all and just copied all from main.ts to main.test.ts to pass the test i think. 
Will do the change. thanks.